pom.xml

```diff
@@ -280,6 +278,7 @@
     <mongo.docker.version>5.0.26</mongo.docker.version>
     <mongodb.version>5.1.4</mongodb.version>
     <netty.version>4.2.0.Final</netty.version>
+    <njord.version>0.5.5</njord.version>
```

Comment:
Update to 0.6.2

Reply:
fixed with 986016e90a56087a3caa11597c929b76c6365cb0